pkg/deployment/apm/plan.go

```diff
@@ -24,12 +24,12 @@ import (
 	"github.com/elastic/cloud-sdk-go/pkg/api"
 	"github.com/elastic/cloud-sdk-go/pkg/client/clusters_apm"
 	"github.com/elastic/cloud-sdk-go/pkg/models"
-	"github.com/elastic/cloud-sdk-go/pkg/plan"
+	"github.com/elastic/cloud-sdk-go/pkg/plan/planutil"
```

Comment:
with the `cmdutil` packages we're setting the aliases as `sdkcmdutil` and `cmdutil`. I think we should do the same for the `planutil` packages. Setting them as `sdkplanutil` and `planutil`. What do you think?

Reply:
Since we've to refactor all of that as part of the `/clusters` service, we might move some of the package to the SDK directly.

Reply:
👌